Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/exchange/transform/uddi/UDDITransform.java

```diff
@@ -298,4 +299,14 @@ private static String buildRole(Contact contact) {
         }
         return null;
     }
+
+    private static List<String> buildDescritpion(BusinessEntity entity) {
+        List<String> descList = new ArrayList<>();
+        if (null != entity && CollectionUtils.isNotEmpty(entity.getDescription())) {
```

Comment:
I feel the isNotEmpty() check is not needed here. The for loop will take care of this case.